source/community.applications/usr/local/emhttp/plugins/community.applications/skins/Narrow/styles/community.applications.css

```diff
@@ -0,0 +1,2392 @@
+  }
+  
+  /* Icon and Symbol Styles */
+  .appDocker,
+  .appLanguage,
+  .appDriver,
+  .appPlugin,
+  .appRepository {
+    float: right;
+    font-size: 2rem;
+    opacity: 0.7;
+    margin-left: 1rem;
+  }
+  
+  .appDocker::before {
+    font-family: 'Unraid';
+    content: '\e90b';
+  }
+  
+  .appLanguage::before {
+    font-family: 'Unraid';
+    content: '\e987';
+  }
+  
+  .appDriver::before {
+    content: '\f2db';
+    font-family: fontAwesome;
+  }
+  
+  .appPlugin::before {
+    font-family: 'Unraid';
+    content: '\e986';
+  }
+
+  /* Menu and Navigation */
+  ul.caMenu,
+  ul.nonselectMenu {
+    list-style-type: none;
+    margin: 0 0 20px 0;
+    padding: 0;
+    font-size: 1.5rem;
+  }
+  
+  li.caMenuItem {
+    padding: 0 0 5px 0;
+    width: fit-content;
+  }
+  
+  .caMenuDisabled {
+    cursor: default;
+    opacity: 0.5;
+  }
+  
+  .caMenuEnabled {
+    cursor: pointer;
+    opacity: 1;
+  }
+
+  .ca_NoAppsFound,
+  .ca_NoDockerAppsFound {
+    font-size: 3rem;
+    margin: auto;
+    text-align: center;
+  }
+  
+  /* Base link styles */
+  
+  .additionalRequirements {
+    margin-left: 2rem;
+  }
+  
+  .additionalRequirementsHeader {
+    font-size: 1.5rem;
+    color: #d67777;
+    font-weight: bold;
+  }
+  
+  .appIconsPopUp {
+    font-size: 2rem !important;
+    cursor: pointer;
+    padding-left: .5rem;
+    padding-right: .5rem;
+    color: default;
+    display: inline-block;
+    margin-right: 5rem;
+  }
+  
+  td.appIconsPopUp {
+    white-space: normal;
+    height: unset;
+  }
+  
+  .appIconsPopUp:hover {
+    text-decoration: none;
+    color: var(--unraid-66-color);
+  }
+  
+  a.appIconsPopUp {
+    text-decoration: none;
+    color: inherit;
+  }
+  
+  .appRepository::before {
+    font-family: 'fontAwesome';
+    content: '\f2be';
+  }
+  
+  .back_to_top_hide {
+    z-index: 0;
+  }
+  
+  .betaCardBackground {
+    clip-path: polygon(0 0,100% 0, 100% 100%);
+    background-color: var(--unraid-66-color);
+    top: 0px;
+    height: 9rem;
+    width: 9rem;
+    position: relative;
+    left: -10rem;
+    margin-right: -9rem;
+  }
+  
+  .betaPopupBackground {
+    clip-path: polygon(0 0,100% 0, 100% 100%);
+    background-color: var(--unraid-66-color);
+    top: var(--beta-popup-offset);
+    height: 9rem;
+    width: 9rem;
+    position: absolute;
+    right: 0;
+  }
+  
+  .betaPopupText {
+    position: absolute;
+    transform: rotate(45deg);
+    -webkit-transform: rotate(45deg);
+    -moz-transform: rotate(45deg);
+    -o-transform: rotate(45deg);
+    color: white;
+    font-size: 2rem;
+    top: 2.25rem;
+    right: -1rem;
+    width: 100%;
+    overflow: hidden;
+    height: 2.5rem;
+  }
+  
+  .caChangeLog {
+    cursor: pointer;
+  }
+  
+  .caChart {
+    height: 1px;
+    width: 3px;
+    border: none;
+  }
+  
+  .caHomeSpotlight {
+    height: 28rem !important;
+  }
+
+
+  .ca_author {
+    font-size: 1rem;
+    margin-left: 0.75rem;
+    margin-top: 0.75rem;
```

Comment:
Nitpick - in the future let's try to get something to automatically organize your rules for you.

It's often best to follow the natural flow of CSS and organize rules similar to how the native `margin` / `padding` rule would - TRBL aka top, right, bottom, left.